homeassistant/components/transport_nsw/sensor.py

```diff
@@ -121,6 +124,11 @@ def update(self) -> None:
         self._icon = ICONS[self._times[ATTR_MODE]]
 
 
+def _get_value(value):  # noqa: N805
```

Comment:
Why are we silencing flake, can we just remove it?
```suggestion
def _get_value(value):
```

Reply:
Thank you. Done